tx_json.go

```diff
@@ -0,0 +1,457 @@
+package wavelet
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/perlin-network/wavelet/sys"
+	"github.com/pkg/errors"
+	"github.com/valyala/fastjson"
+)
+
+// Parser defines a generic JSON transaction payload parser.
+type Parser struct {
```

Comment:
Possible to rename to `TransactionParserJSON`?

Reply:
Yup. Made more sense when it was just in its own package in cmd/wctl. Forgot to change it when it was moved to the root.

Reply:
But yeah, totally agree.